src/modules/shortcut_guide/overlay_window.cpp

```diff
@@ -594,7 +614,7 @@ void D2DOverlayWindow::render(ID2D1DeviceContext5* d2d_dc) {
   }
   // render the monitors
   if (render_monitors) {
-    brushColor = D2D1::ColorF(colors.desktop_fill_color, minature_shown ? current_anim_value : current_anim_value * 0.3f);
+    brushColor = D2D1::ColorF(colors.desktop_fill_color, 1.0f);
```

Comment:
I'm -1 on this change. This makes the monitor preview faded out a little, just as the arrow keys when no window is selected.

Reply:
Before I made any changes to this and other opacity related operations, the preview was faded out in the same way when no window was selected.

Reply:
It is for the "dimmed" monitors when no app is active, like here: https://ibb.co/0D9XNm3

Reply:
Fixed in https://github.com/microsoft/PowerToys/pull/521/commits/5313c7d715284d0dda537c6a55a89b66590a65c9